### PR TITLE
Use the latest versions we have of the system packages

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request modifies the Ansible role to use the latest versions we have of the system packages (and GPG signing key for RPMs) for the Crowdstrike agent.

## 💭 Motivation and context ##

We should stay current.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dv4harr10 — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
🦅

File: vars/Amazon.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
 # The name of the S3 object corresponding to the CrowdStrike Falcon
 # sensor system package
-package_object_name: falcon-sensor-6.56.0-15309.amzn2023.x86_64.rpm
+package_object_name: falcon-sensor-7.05.0-16004.amzn2023.x86_64.rpm
+
+# The name of the S3 object corresponding to the GPG key for the
+# CrowdStrike Falcon system package
+package_gpg_key_object_name: "Falcon Linux Sensor RPM signing GPG key 2023.gpg"
```
Comment (mcdonnnj) [inline vars/Amazon.yml]: Are the quotes necessary or desired?

```suggestion
package_gpg_key_object_name: Falcon Linux Sensor RPM signing GPG key 2023.gpg
```

File: vars/RedHat.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 # The name of the S3 object corresponding to the CrowdStrike Falcon
 # sensor system package
-package_object_name: falcon-sensor-6.54.0-15110.el8.x86_64.rpm
+package_object_name: falcon-sensor-7.06.0-16108.el9.x86_64.rpm
 
 # The name of the S3 object corresponding to the GPG key for the
 # CrowdStrike Falcon system package
-package_gpg_key_object_name: "Falcon Linux Sensor RPM signing GPG key"
+package_gpg_key_object_name: "Falcon Linux Sensor RPM signing GPG key 2023.gpg"
```
Comment (mcdonnnj) [inline vars/RedHat.yml]: Same question about quotes.

```suggestion
package_gpg_key_object_name: Falcon Linux Sensor RPM signing GPG key 2023.gpg
```

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ with one pair of suggestions for your consideration.

**Pull request merged**